pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -501,44 +501,66 @@ func buildInboundCatchAllHTTPFilterChains(configgen *ConfigGeneratorImpl,
 			Bind:                       matchingIP,
 			InboundClusterName:         clusterName,
 		}
+		var allChains []istionetworking.FilterChain
```

Comment:
can you add a comment on the PR about why all of this is changing? I would expect only `false` -> `true` here, not code changes

Reply:
The previous code doesn't call the plugin to install the authn/authz policy and we need to add code here to call plugins.

Reply:
I will add comment.

Reply:
Document added. Thanks!